Assets/Scripts/Enemy/Minion/MinionController.cs

```diff
@@ -1,18 +1,20 @@
 using System.Collections;
 using System.Collections.Generic;
+using Unity.VisualScripting;
 using UnityEngine;
 
 public class MinionController : MonoBehaviour
 {
     //Basic properties and component refs...
     public GameObject anchor;
     public GameObject spawner;
-    public float health = 20F;
+    public HealthBar healthBar;
+    public float maxhealth = 500;
     public float moveSpeed = 3F;
     public float collisionDmg = 5F;
     private bool awayFromAnchor = true;
     private Rigidbody2D rb;
-    
+
     //Parameters describing the minion's attack properties...
     [SerializeField] private GameObject projectile;
     public AttackPattern firingPattern = AttackPattern.FixedAlternating;
```

Comment:
I don't believe these are actually being used anywhere, are they?
Let's remove them so it doesn't confuse anyone (eg, game designers trying to tweak this from the Unity editor)